Validation/src/Validate/ecal.py

```diff
@@ -110,6 +110,35 @@ def veto_results(d : Differ, out_dir = None) :
         log.info(f'plotting {col}')
         d.plot1d(col, name, out_dir = out_dir)
 
+@plotter(hist=True,event=False)
+def clue_cluster(d : Differ, out_dir = None) :
+    """
+    Plot ECal CLUE cluster features from the already created DQM histograms
+    Parameters
+    ----------
+    d : Differ
+        Differ containing files that are not event files (presumably histogram files)
+    """
+
+    col, name = 'EcalClusterAnalyzer/EcalClusterAnalyzer_number_of_clusters', 'Number of CLUE clusters'
+    log.info(f'plotting {col}')
+    d.plot1d(col, name, out_dir = out_dir, legend_kw = dict(loc='upper left'))
+
+    features = [
+        ('EcalClusterAnalyzer/EcalClusterAnalyzer_number_of_clusters', 'Number of CLUE clusters'),
+        ('EcalClusterAnalyzer/EcalClusterAnalyzer_correctly_predicted_events', 'Correctly predicted events'),
```

Comment:
Change the axis labels (like the PN event types) so that the plot says "under", "correct", "over" instead of numeric bin labels.
You can also update the bin labels in the C++ side so they show up in ROOT's presentation (again, see the PN DQM for example).